serverlessworkflow/modules/ROOT/pages/getting-started/modules/proc-running-application.adoc

```diff
@@ -0,0 +1,145 @@
+:_content-type: PROCEDURE
+:description: Running your workflow application
+:keywords: run, serverless, workflow, application
+:navtitle: Running your workflow application
+// :page-aliases:
+
+[id="proc-running-application_{context}"]
+= Running your workflow application
+
+After creating a workflow, you can run your workflow application.
+
+.Procedure
+. Enter the following command to run your workflow application:
++
+
+[tabs]
+====
+Quarkus CLI::
++
+--
+.Run your workflow application using Quarkus CLI
+[source,shell]
+----
+quarkus dev
+----
+--
+Apache Maven::
++
+--
+.Run your workflow application using Apache Maven
+[source,shell]
+----
+mvn clean quarkus:dev
+----
+--
+Knative workflow CLI:
++
+--
+Currently, Knative workflow CLI doesn't support yet running workflows locally. You can either:
+.Run your workflow application using Apache Maven
+[source,shell]
+----
+mvn clean quarkus:dev
+----
+.Run your workflow application using Quarkus CLI
+[source,shell]
+----
+quarkus dev
+----
+For more information about Knative workflow CLI, see xref:tooling/kn-plugin-workflow-overview.adoc[{context} plug-in for Knative CLI].
+Also, to deploy and run your workflow application, see xref:cloud/deploying-on-minikube.adoc[Deploying workflow application on Minikube]
+--
+====
```

Comment:
This part of tabs is not correctly rendered. This was also present before but maybe we can fix it now. The issue is probably a missing second colon, so it should be `Knative workflow CLI::`